modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -25,6 +25,14 @@ components:
                     type: object
                 oldValue:
                     type: object
+        Error:
```

Comment:
I would call this `ValidateError` to keep consistency with the other schemas. Also `Error` is too generic and could difficult modeling different errors in the future.